utils/index.d.ts

```diff
@@ -169,15 +169,15 @@ export interface AzExtServiceClientCredentialsT2 {
      * @param options - The options used to configure any requests this
      *                TokenCredential implementation might make.
      */
-    getToken(scopes?: string | string[], options?: any): Promise<any | null>;
```

Comment:
Shouldn't we keep `options?` even though we aren't using it? The `TokenCredential` interface has it in its `getToken()` method. https://learn.microsoft.com/en-us/javascript/api/@azure/identity/tokencredential?view=azure-node-latest

Reply:
Sure